lib/rules/color-no-out-gamut-range/README.md

```diff
@@ -0,0 +1,78 @@
+# color-no-out-gamut-range
+
+Disallow out of gamut range colors.
+
+<!-- prettier-ignore -->
+```css
+a { color: lch(48% 82 283) }
+/**        ↑
+ * This color is out of sRGB gamut range */
+```
+
+Colors declared using lch, oklch, lab and oklab functions can move from sRGB to P3 color space, which is not supported by most of screens right now. It can happen by mistake. For instance, by converting colors from one space to another.
+
+This rule checks if the color is in sRGB space. If it's not, you should wrap it in media query like this:
+
+```css
+@media (color-gamut: p3) {
+  a {
+    color: lch(48% 82 283);
+  }
+}
+```
+
```

Comment:
[suggest] Could you please address the following comment in https://github.com/stylelint/stylelint/issues/5905#issuecomment-1043244183?

> We'll need to document the limitation that conditional imports won't be respected, e.g.: